src/script/cpp_api/s_security.cpp

```diff
@@ -457,11 +457,10 @@ bool ScriptApiSecurity::safeLoadFile(lua_State *L, const char *path, const char
 	size_t start = 0;
 	int c = std::getc(fp);
 	if (c == '#') {
-		// Skip the first line
-		while ((c = std::getc(fp)) != EOF && c != '\n') {}
-		if (c == '\n')
-			std::getc(fp);
-		start = std::ftell(fp);
+		// Skip the content of the first line
```

Comment:
Maybe “skip the shebang line” if that’s the intention?
Also the shebang line always starts with `#!` AFAIK, not just `#`.

Reply:
The old code accepted `#`, so I've kept that.

I've changed the comment.